src/main/java/net/pms/util/SubtitleUtils.java

```diff
@@ -639,7 +639,7 @@ public static InputStream removeSubRipTags(File file) throws IOException {
 		BufferedReader input = FileUtil.bufferedReaderWithCorrectCharset(file);
 		ByteArrayOutputStream os = new ByteArrayOutputStream();
 		Writer writer = new OutputStreamWriter(os, Charset.forName(CHARSET_UTF_8));
-		Pattern pattern = Pattern.compile("\\</?(?:b|i|s|u|font[^\\>]*)\\>|\\{\\\\[^\\}]+\\}|\\\\h|\\\\N");
+		Pattern pattern = Pattern.compile("\\</?(?:b|i|s|u|font[^\\>]*)\\>|\\{\\\\.*?}|\\\\h|\\\\N");
```

Comment:
@Nadahar of course I know that the resullt will be the same but I think that to use `.*?` is a little bit more general to say that any characters between `{\...}` should be replaced. It is about the preferences so feel free to revert this commit.